actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsActor.java

```diff
@@ -213,216 +224,206 @@ private void courseProgressMetrics(Request actorMessage) {
-        sender().tell(response, self());
-        return;
-      } else {
+      Response response = new Response();
+      response.putAll(responseMap);
+      sender().tell(response, self());
+      return;
+    } else {
 
-        ProjectLogger.log(
-            "CourseMetricsActor-courseProgressMetrics-- Courses for batch is empty .");
+      ProjectLogger.log(
+          "CourseMetricsActor:courseProgressMetrics courses for batch is empty.",
+          LoggerEnum.INFO.name());
 
-        Map<String, Object> responseMap = new LinkedHashMap<>();
-        Map<String, Object> userdataMap = new LinkedHashMap<>();
-        Map<String, Object> courseprogressdataMap = new LinkedHashMap<>();
-        Map<String, Object> valueMap = new LinkedHashMap<>();
+      Map<String, Object> responseMap = new LinkedHashMap<>();
+      Map<String, Object> userdataMap = new LinkedHashMap<>();
+      Map<String, Object> courseprogressdataMap = new LinkedHashMap<>();
+      Map<String, Object> valueMap = new LinkedHashMap<>();
 
-        userdataMap.put(JsonKey.NAME, "List of users enrolled for the course");
-        userdataMap.put("split", "content.sum(time_spent)");
-        userdataMap.put("buckets", new ArrayList<>());
+      userdataMap.put(JsonKey.NAME, "List of users enrolled for the course");
+      userdataMap.put("split", "content.sum(time_spent)");
+      userdataMap.put("buckets", new ArrayList<>());
 
-        courseprogressdataMap.put(JsonKey.NAME, "List of users enrolled for the course");
-        courseprogressdataMap.put("split", "content.sum(time_spent)");
-        courseprogressdataMap.put("buckets", esContent);
+      courseprogressdataMap.put(JsonKey.NAME, "List of users enrolled for the course");
+      courseprogressdataMap.put("split", "content.sum(time_spent)");
+      courseprogressdataMap.put("buckets", esContent);
 
-        valueMap.put("course.progress.users_enrolled.count", userdataMap);
-        valueMap.put("course.progress.course_progress_per_user.count", courseprogressdataMap);
+      valueMap.put("course.progress.users_enrolled.count", userdataMap);
+      valueMap.put("course.progress.course_progress_per_user.count", courseprogressdataMap);
 
-        responseMap.put("period", periodStr);
-        responseMap.put("series", valueMap);
+      responseMap.put("period", periodStr);
+      responseMap.put("series", valueMap);
 
-        Response response = new Response();
-        response.putAll(responseMap);
-        cache.setData(JsonKey.CourseProgress, batchId, periodStr, responseMap);
-        sender().tell(response, self());
+      Response response = new Response();
+      response.putAll(responseMap);
+      sender().tell(response, self());
 
-        return;
-      }
+      return;
     }
   }
 
   private void courseConsumptionMetrics(Request actorMessage) {
-    ProjectLogger.log("In courseConsumptionMetrics api");
+    ProjectLogger.log(
+        "CourseMetricsActor:courseConsumptionMetrics consumption call start.",
```

Comment:
From:
"CourseMetricsActor:courseConsumptionMetrics consumption call start."

To:
"CourseMetricsActor: courseConsumptionMetrics called."